README.rst

```diff
@@ -1,84 +1,138 @@
-To determine what realm you are in, check your profile page in the SignalFx
-web application (click the avatar in the upper right and click My Profile).
+**Option 2: Create a Lambda function, then attach the SignalFx-hosted layer**
 
+In this option, you will use a layer created and hosted by SignalFx.
 
-Environment Variables
-~~~~~~~~~~~~~~~~~~~~~
+1. To verify compatibility, review the list of supported regions. See [Lambda Layer Versions](https://github.com/signalfx/lambda-layer-versions/blob/master/python/PYTHON.md).
+2. Open your AWS console. 
+3. In the landing page, under **Compute**, click **Lambda**.
+4. Click **Create function** to create a layer with SignalFx's capabilities.
+5. Click **Author from scratch**.
+6. In **Function name**, enter a descriptive name for the wrapper. 
+7. In **Runtime**, select the desired language.
+8. Click **Create function**. 
+9. Click on **Layers**, then add a layer.
+10. Mark **Provide a layer version**.
+11. Enter an ARN number. 
+  * To locate the ARN number, see [Lambda Layer Versions](https://github.com/signalfx/lambda-layer-versions/blob/master/python/PYTHON.md).
 
-**Note: the environment variables ``SIGNALFX_INGEST_ENDPOINT`` and ``SIGNALFX_AUTH_TOKEN`` are being deprecated and will not be supported in future releases.**
+~~~~~
 
-.. code:: bash
+**Option 3: Create a Lambda function, then create and attach a layer based on a SignalFx template**
 
-    SIGNALFX_ACCESS_TOKEN=access token
+In this option, you will chose a SignalFx template, and then deploy a copy of the layer.
+
+1. Open your AWS console. 
+2. In the landing page, under **Compute**, click **Lambda**.
+3. Click **Create function** to create a layer with SignalFx's capabilities.
+4. Click **Browse serverless app repository**.
+5. Click **Public applications**.
+6. In the search field, enter and select **signalfx-lambda-python-wrapper**.
+7. Review the template, permissions, licenses, and then click **Deploy**.
+    * A copy of the layer will now be deployed into your account.
+8. Return to the previous screen to add a layer to the function, select from list of runtime compatible layers, and then select the name of the copy.  
+
+
+~~~~~
+
+Step 2: Update the SignalFx realm
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, this function wrapper will send data to the ``us0`` realm. If you are not in this realm, and you are not using the gateway, then you will need to update the realm for the SignalFx subdomain, such as us1.signalfx.com ( {REALM}.signalfx.com  ). 
+
+To locate your realm:
+
+1. Open SignalFx and in the top, right corner, click your profile icon.
+2. Click **My Profile**.
+3. Next to **Organizations**, review the listed realm.
+
+Additionally, you must adjust the variables you set to the correct realm ingest endpoint (``https://ingest.{REALM}.signalfx.com``). These variables include ``SIGNALFX_ENDPOINT_URL``, ``SIGNALFX_TRACING_URL``, ``SIGNALFX_METRICS_URL``, and ``SIGNALFX_INGEST_ENDPOINT``. 
```

Comment:
I would remove line 82. With the all possible setups it is probably more confusing than helping.  I think it will be clear that if they want to use ingest directly, they need to account for the realm.

Reply:
This has been removed.